### ISSUE TITLE
The Keyboard shown at the bottom of the screen is wrong. Shows C5 for C4

### ISSUE THREAD

**Issue by masoudd** [leafo/sightreading.training#44]
The note in the middle of the grand staff is C4 not C5.

**Comment by aik099** [comment]
No standard for that. That depends on an app:

* in GarageBand C3 is Middle C
* in Synthesia C4 is Middle C

I agree, that for full sized piano keyboard the `C4` being `Middle C` is more common.

**Comment by fedorl** [comment]
> No standard for that
@aik099  There IS a standard for what is C4 or C5 for usual piano tuning, though. When you press a key labeled C5 on the site keyboard and it plays piano C4 sound instead, it sounds wrong and is not a matter of opinion. It is also wrong when the lowest  note in settings dropdown shows A4, keyboard label counts A4, and the app **actually** plays the A3 piano sound (and on the standard treble clef it is indeed A3).

**Comment by Cleroth** [comment]
Agreed with @fedorl . I don't see what's "controversial" here. There is a piano shown, most people training on this would likely be using a piano, so the 'first' C on a treble clef should be C4, not C3, and certainly not C5.
Mostly it makes it really confusing when selecting note ranges.
